### PR TITLE
version is tracked in one place only; closes #54

### LINKED ISSUE #54: Try to go without bumpversion
And keep the package version in one place only, in top level `__init__.py`.

### PR DESCRIPTION
**Pull request by bast**
With this we do not need bumpversion anymore.

### CODE REVIEW

**Review by robertodr — APPROVED** [review]
Bump it off the wagon!

File: setup.py [hunk 1]
```diff
@@ -3,13 +3,21 @@
...
 with open("README.rst") as readme_file:
     readme = readme_file.read()
 
 with open("HISTORY.rst") as history_file:
     history = history_file.read()
 
+# extract __version__ from __init__.py
+_here = os.path.abspath(os.path.dirname(__file__))
+_version = {}
+with open(os.path.join(_here, 'parselglossy', '__init__.py')) as f:
```
Comment (robertodr) [inline setup.py]: Not terribly important, but could you do it with `pathlib`? :smile_cat:
```
with Path("parselglossy/__init__.py").open("r") as f:
       exec(f.read(), _version)
```

**Comment by bast** [comment]
In "Deploying" (step 4), why do we need "Remember to disengage branch protection for the master branch"? I wish we could get this to master via code review. Mistakes happen and I like if somebody else looks over version changes.

File: setup.py (hunk 1 above)
Comment (bast) [reply setup.py]: I was thinking about it and was wondering whether this would somehow limit us but I think it is a good idea. I will modernize that.

**Comment by robertodr** [comment]
I wrote that because I am not aware of a way to tag by pull request.

**Comment by bast** [comment]
But how about we submit as PRs and once it's on master, we tag then? Does one need to disengage branch protection to push a tag?

**Comment by robertodr** [comment]
I think so? Can you tag from the GitHub UI? If yes, then I completely agree to do the last commit(s) before release with pull requests and the tag clickety-click _via_ the web interface.

**Comment by bast** [comment]
Yes - I would go for that, I will adjust the guide. I think this is safer.

**Comment by bast** [comment]
Yes, I typically these days tag from the UI.

**Comment by robertodr** [comment]
Just tried (minted `v0.3.0-alpha1`) I was not aware of that. So much safer and easier! It also encourages you to actually write a short release summary, which is nice!

**Comment by bast** [comment]
Yes - I really like these brief summaries. Like a mini-changelog. Really nice for users.

**Pull request merged**